app/assets/javascripts/oxalis/api/api_latest.js

```diff
@@ -776,8 +776,8 @@ class UtilsApi {
    * // removeToast();
    */
   showToast(type: ToastStyleType, message: string, timeout: number): ?Function {
-    const noop = () => {};
-    return Toast.message(type, message, timeout === 0, timeout).remove || noop;
+    Toast.message(type, message, timeout === 0, timeout);
+    return Toast.close(message);
```

Comment:
This will immediately close the Toast, you probably wanted to return `() => Toast.close(message)`.

Reply:
well spotted